hack/release/prepare-assets.sh

```diff
@@ -104,7 +104,8 @@ cp ./hack/externalnode/install-vm.ps1 "$OUTPUT_DIR/"
 
 export IMG_TAG=$VERSION
 
-export IMG_NAME=projects.registry.vmware.com/antrea/antrea-ubuntu
```

Comment:
Please check if all `IMG_NAME` can still work with generate-manifest.sh in this file.

Reply:
yes it works.

Reply:
Given that you have removed IMG_NAME in generate-manifest.sh, why it still can work? Could you explain more?

Reply:
by all IMG_NAME works i mean to say that AGENT_IMG_NAME and CONTROLLER_IMG_NAME works fine, IMG_NAME is removed so it won't work with generate-manifest.sh

Reply:
Got your point, thanks for the explanation.